packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -1,21 +1,17 @@
 import React, { ReactElement, useState } from 'react';
-import classNames from 'classnames';
-import { ModalProps, StyledModal } from '../modals/StyledModal';
-import SidebarList from '../sidebar/SidebarList';
+import { ModalProps } from '../modals/StyledModal';
 import HashtagIcon from '../icons/Hashtag';
 import FilterIcon from '../icons/Filter';
 import BlockIcon from '../icons/Block';
 import TagsFilter from './TagsFilter';
 import { TagCategoryLayout } from './TagCategoryDropdown';
 import AdvancedSettingsFilter from './AdvancedSettings';
 import BlockedFilter from './BlockedFilter';
-import CloseButton from '../CloseButton';
-import styles from './FeedFilters.module.css';
```

Comment:
I think this was the last use of this, could you double check and remove this file?